src/spring/HISTORY.md

```diff
@@ -1,5 +1,13 @@
 Release History
 ===============
+4.0.0
+---
+* Rename extension name to "Spring".
+
```

Comment:
Since `spring` is actually a new extension, do we need to release this extension after the latest version `3.1.6` of original extension? 
In fact, I think the version for `spring` and version for `spring-cloud` can be released independently. Otherwise, the `spring` extension has only the version after version `4.0.0`, and there is no version before `4.0.0`. And these two extensions will coexist for some time in the future

Reply:
In addition, I think the renaming of extension can be described in the history note of `sping-cloud` extension, but it may not need to be described in `sping` extension. Because the users using the `spring` extension may be new users and do not need to care about its original extension `sping-cloud`.

Reply:
updated